build-edt.sh

```
@@ -32,14 +32,14 @@ edt_version=$EDT_VERSION
 edt_escaped="${edt_version// /_}"
 
 docker build \
-        --pull \
+    --pull \
     $no_cache_arg \
```

Shellcheck (0.11.0)

[warning] no_cache_arg is referenced but not assigned.

(SC2154)